setup.py

```diff
@@ -54,13 +55,17 @@
     dependency for dependency in open("requirements_gis.txt").readlines()
 ]
 # Special GDAL handling
-try:
-    gdal_version = subprocess.run(
-        ["gdal-config", "--version"], capture_output=True
-    ).stdout.decode("utf-8")
-    gis_requirements.append(f"gdal=={gdal_version}")
-except subprocess.CalledProcessError:
-    pass
+on_conda = os.getenv("CONDA_BUILD")
```

Comment:
I would clarify the truthiness of this expression, by casting to a `bool` or something, because as is, it seems dangerous to me (i.e. `"0"` is truthy).

Reply:
Does `if on_conda is None` work for us?

Reply:
`os.getenv` returns the value as a string, so if it is anything but an empty string, negating it with `not` will not do what you want I think.

Reply:
Gotcha. Let's be very specific then.